recipes/boost/all/conanfile.py

```diff
@@ -642,14 +622,15 @@ def _build_cross_flags(self):
         if arch.startswith('arm'):
             if 'hf' in arch:
                 flags.append('-mfloat-abi=hard')
+        elif self.settings.os == "Emscripten":
+            flags.append('linkflags="-c"')
```

Comment:
what does -c mean and why is it required?

Reply:
It was mainly to resolve warnings of type:
```
em++: warning: Assuming object file output in the absence of `-c`, based on output filename. Add with `-c` or `-r` to avoid this warning [-Wemcc]
```
It tells the compiler to generate LLVM bitcode (https://emscripten.org/docs/tools_reference/emcc.html)

Is it wrong?

Reply:
Practically it is not necessary and they can be removed (and I think they did not remove completely the warnings, but it just reduced them).